src/components/ui/Badge.tsx

```diff
@@ -1,15 +1,15 @@
 import { type VariantProps, cva, cx } from '@/lib/utils';
 
 const badgeVariants = cva({
-  base: 'inline-flex items-center rounded-md border px-2.5 py-0.5 font-semibold text-xs transition-colors focus:outline-none focus:ring-2 focus:ring-ring focus:ring-offset-2',
+  base: 'inline-flex items-center rounded-md border px-2.5 py-0.5 font-semibold text-xs transition-colors focus:outline-hidden focus:ring-2 focus:ring-ring focus:ring-offset-2',
```

Comment:
The border is now for some reason visible and looks horrible. Take a look at the events page and the "internal" badge there as an example
```suggestion
  base: 'inline-flex items-center rounded-md px-2.5 py-0.5 font-semibold text-xs transition-colors focus:outline-hidden focus:ring-2 focus:ring-ring focus:ring-offset-2',
```